modules/ocf_irc/files/anope.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=A set of IRC Services
+After=network-online.target inspircd.service
+
+[Service]
+User=irc
+Group=irc
+Restart=always
+# Run ExecStartPre with root permissions to be able to create pid directory
+PermissionsStartOnly=true
+ExecStartPre=/bin/mkdir -p /var/run/anope/
+ExecStartPre=/bin/chown -R irc:irc /var/run/anope/
```

Comment:
Yeah, that seems reasonable, although I think it is more readable currently, especially if someone doesn't know what `install` does, which is pretty common... That being said it's a cleaner command to have instead of two separate ones to do the same thing.